src/libcore/macros.rs

```diff
@@ -229,14 +229,26 @@ macro_rules! try {
     })
 }
 
-/// Use the `format!` syntax to write data into a buffer.
+/// Calls `write_fmt` function on a writer
```

Comment:
That looks much better. Thank you.

I was already aware that it needed to be as short as possible, since I compiled the docs and experienced it.